Makefile

```diff
@@ -36,6 +36,8 @@ dockerbuild: my.env
 # postgres going forward and has the needed environment variables.
 dockersetup: my.env .docker-build
 	${DC} run webapp /app/docker/run_setup_postgres.sh
+	${DC} run processor /app/docker/run_recreate_s3_buckets.sh
+	${DC} run processor /app/scripts/socorro clear_es_indices
```

Comment:
The reason that the first item runs in the webapp container is that we were thinking that the webapp would ultimately own all the data stores. That's a vision change related to the processor rewrite.

Figured I'd mention it in case that's interesting to know.

The other thought I had here is that it's "faster" if all three of these scripts are executed in a single `docker-compose run` rather than three separate ones. It's probably a second or two in savings, so it's probably not a big deal. If there are other niceties to running all three of these scripts as a single script, then maybe it's worth changing.